src/controller/java/BUILD.gn

```diff
@@ -369,12 +371,20 @@ kotlin_library("kotlin_matter_controller") {
     "src/matter/controller/CompletionListenerAdapter.kt",
     "src/matter/controller/ControllerParams.kt",
     "src/matter/controller/InteractionClient.kt",
+    "src/matter/controller/InvokeCallback.kt",
+    "src/matter/controller/InvokeCallbackJni.kt",
     "src/matter/controller/MatterController.kt",
     "src/matter/controller/MatterControllerException.kt",
     "src/matter/controller/MatterControllerImpl.kt",
     "src/matter/controller/Messages.kt",
     "src/matter/controller/OperationalKeyConfig.kt",
+    "src/matter/controller/ReportCallback.kt",
+    "src/matter/controller/ReportCallbackJni.kt",
+    "src/matter/controller/ResubscriptionAttemptCallback.kt",
+    "src/matter/controller/SubscriptionEstablishedCallback.kt",
     "src/matter/controller/SubscriptionStates.kt",
+    "src/matter/controller/WriteAttributesCallback.kt",
+    "src/matter/controller/WriteAttributesCallbackJni.kt",
     "src/matter/controller/model/Paths.kt",
     "src/matter/controller/model/States.kt",
   ]
```

Comment:
Need to remove ":java" dependence from kotlin_matter_controller

Reply:
No. we can't. The dependency of ChipDeviceController has not been completely removed yet. After this PR is merged, separation work is required for the remaining API.

Reply:
Acked.